collections/_infographics/emissions/fossil-fuels-emissions.md

```diff
@@ -1,16 +1,60 @@
 data-our:      "https://docs.google.com/spreadsheets/d/19PiqYV6FkttzrNcjkbTQmphzxgydtTc1MB6nDZpybj0/edit?usp=sharing"
 data-orig:
   - [ "Global Carbon Budget", "https://www.icos-cp.eu/science-and-impact/global-carbon-budget/2020" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emise-fosilni-paliva" %}
+## How to read this graph?
+
+* The graph shows **worldwide emissions from burning coal, oil and natural gas** and, for comparison, **emissions from cement production**. Total emissions from these sources for 2019 were 35,897 million tons of CO<sub>2</sub>.
+* The graph only shows emissions from the chemical reaction CaCO<sub>3</sub> + heat ⟶ CaO + CO<sub>2</sub> as emissions from cement production. Emissions created by burning coal or gas for heating are counted as emissions of the respective fuel.
+
+## More about individual fuels and cement
+
+{% comment %}
+{% include preview-box.html
+    title="What is electricity produced from?"
+    text="Unlike oil, coal and natural gas are important fuels for electricity generation."
+    slug="elektrina-svet"
+%}
+{% endcomment %}
+
+### Coal
+
+* **Usage:** Primarily for the production of electricity and heat ([Mapped worlds coal power plants](https://www.carbonbrief.org/mapped-worlds-coal-power-plants)) in the iron or glass smelting industry or other heating.
+* **Mining and Transportation:** In 2019, the world mined approximately [8 billion tons of coal](https://www.iea.org/reports/coal-information-overview). China (46% of world production) mined by far the most coal, followed by India (10%), the USA (8%), Indonesia (8%) and Australia (6%). Countries need to import coal to the point of consumption, unlike pipelines which help transport oil and natural gas.
+* The graph shows a remarkable **increase in coal consumption after 2000** caused by the industrialization of China, India and other Asian countries. Coal consumption in Europe and the USA has been declining in recent decades.
+
+### Petroleum
+
+* **Usage:** Most of the oil – or, more accurately, the gasoline and diesel produced from oil – is burned in transportation. It is used for the production of electricity and heat rather exceptionally.
+* **Extraction and transportation:** The world produces about [100 million barrels of oil](https://www.eia.gov/outlooks/steo/report/global_oil.php) daily, or about 5 billion tons per year (2019). Most oil is produced in the countries of the Persian Gulf (about 32% of world production), followed by the USA (15%) and Russia (13%). While the US covers its consumption with production, most other major oil producers export their production. Ships [transport about 61% of crude oil by sea](https://talkbusiness.net/2017/08/61-of-global-crude-oil-and-petroleum-products-transported-by-sea/) and the rest through [ pipelines](http://worldmap.harvard.edu/data/geonode:global_oil_pipelines_7z9).
```

Comment:
Ships [transport about 61% of crude oil by sea](https://talkbusiness.net/2017/08/61-of-global-crude-oil-and-petroleum-products-transported-by-sea/) and the rest through [ pipelines](http://worldmap.harvard.edu/data/geonode:global_oil_pipelines_7z9).

Approximately 61% of crude oil is transported by ship [link], with the remainder transported by pipelines [link].

This translation isn't the most exact to the Czech original, but I think "transported by ship" is less meaning-ambiguous than "transported by sea". Sentence structure needed modification.

Reply:
The sentence structure being modified made sense. I also changed the word `sea` to `ship` as suggested but I was not sure to what extend we want to stay true to the original.